### DISCUSSION TITLE
ghippo/permissions/mspider

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2779] [Announcements]
# ghippo/permissions/mspider

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/permissions/mspider.html

**Comment by samzong** [comment]
文档需要更新，补充新功能模块，并进行权限说明.

#2778
